src/shared/components/ui/DateTimePicker/DateTimePicker.js

```diff
@@ -27,7 +26,8 @@ const DateTimePicker = ({
   oneDatePicker,
   selectedDateCalendar,
   showMonthAndYearPickers,
-  i18n
+  i18n,
+  getLanguage
```

Comment:
1. nie dodałeś propsa do opisu
2. w sumie dlaczego chcesz to wynieść?

Reply:
Przemyślałem to i zmieniłem implementację lekko :)

Reply:
zmieniłeś, ale dlaczego? co tutaj będzie lepszego? 
Z mojego punktu widzenia to zmiana na gorsze, wcześniej getLanguage było zamknięte w komponencie, i nie trzeba było go przekazywać, wszystko działało "w tle", tak jak do innych komponentów też nie przekazuję konkretnego języka :) Minusem był dodatkowy helper w teście, ale akurat dla mnie nie jest to nic przeważającego.

Reply:
zerknij prosze jeszcze raz, nie przekazuje już nic w komponencie, zmodyfikowałem helperka i dodałem mu fallback. Z ui-kit trzeba pozabyć się całkowicie tłumaczeń bo z nich nie korzystamy i tylko zajmują miejsce. Dlatego wszystko wywaliłem, początkowo helperka też bo testy się wywalały ale wpadłem na pomysł, że dodam po prostu fallback(i to była ta zmiana implementacji XD)